conf/modules.config

```
@@ -1026,16 +1026,28 @@ process{
             (params.vep_dbnsfp && params.dbnsfp)                                  ? '--plugin dbNSFP,dbNSFP.gz,rs_dbSNP,HGVSc_VEP,HGVSp_VEP,1000Gp3_EAS_AF,1000Gp3_AMR_AF,LRT_score,GERP++_RS,gnomAD_exomes_AF' : '',
             (params.vep_loftee)                                                   ? '--plugin LoF,loftee_path:/opt/conda/envs/nf-core-vep-104.3/share/ensembl-vep-104.3-0'                                      : '',
             (params.vep_spliceai && params.spliceai_snv && params.spliceai_indel) ? '--plugin SpliceAI,snv=spliceai_scores.raw.snv.hg38.vcf.gz,indel=spliceai_scores.raw.indel.hg38.vcf.gz'                     : '',
-            (params.vep_spliceregion)                                             ? '--plugin SpliceRegion'                                                                                                     : ''
+            (params.vep_spliceregion)                                             ? '--plugin SpliceRegion'                                                                                                     : '',
+            (params.vep_out_format)                                               ? "--${params.vep_out_format}"                                                                                                : '--vcf'
         ].join(' ').trim()
         if (!params.vep_cache)    container = { params.vep_genome ? "nfcore/vep:104.3.${params.vep_genome}" : "nfcore/vep:104.3.${params.genome}" }
         publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/reports/EnsemblVEP/${meta.id}/${meta.variantcaller}" },
-            pattern: "*html"
+            [
+                mode: params.publish_dir_mode,
+                path: { "${params.outdir}/reports/EnsemblVEP/${meta.id}/${meta.variantcaller}" },
+                pattern: "*html"
+            ],
+            [
+                mode: params.publish_dir_mode,
+                path: { "${params.outdir}/annotation/${meta.id}/${meta.variantcaller}" },
+                pattern: "*{json,tab}"
+            ]
         ]
     }
 
+    withName: 'NFCORE_SAREK:SAREK:ANNOTATE:ANNOTATION_ENSEMBLVEP:ENSEMBLVEP' {
+        ext.prefix      = {"${meta.id}_VEP"}
+    }
+
     withName: ".*:ANNOTATION_MERGE:ENSEMBLVEP" {
         ext.prefix       = {"${meta.id}_snpEff"}
```

```suggestion
        ext.prefix       = {"${meta.id}_snpEff_VEP"}
```
I think we need to add `_VEP` here too.
Can you check when producing annotated vcf with --tools merge if the `_VEP` is there?